test_runtime.sh

```diff
@@ -77,9 +77,9 @@ trap cleanup EXIT
 tar -xf  rootfs.tar.gz -C ${TESTDIR}
 cp runtimetest ${TESTDIR}
 
-ocitools generate --output "${TESTDIR}/config.json" "${TEST_ARGS[@]}" --rootfs '.'
+ocitools generate --tty --output "${TESTDIR}/config.json" "${TEST_ARGS[@]}" --rootfs '.'
```

Comment:
--tty is unnecessary ,  add it we can see the output info,wouldn't it be better?